inc/patches/3-9-3-1.patch

```diff
@@ -111,3 +111,17 @@ index 976d4eb..155c1f6 100644
  #else
  /******************************************************************************/
  /**
+
+diff --git a/crypto/keyformat/pkcs.c b/crypto/keyformat/pkcs.c
+index f1a725e..afb537f 100644
+--- a/crypto/keyformat/pkcs.c
++++ b/crypto/keyformat/pkcs.c
+@@ -519,7 +519,8 @@ int32 psPkcs8ParsePrivBin(psPool_t *pool, unsigned char *buf, int32 size,
+         if (plen > 0)
+         {
+             /* Unexpected extra data remains. Treat it as an error. */
+-            goto PKCS8_FAIL;
++            psTraceIntCrypto("Unexpected extra data: %d\n", plen);
++            //goto PKCS8_FAIL;
```

Comment:
Are you sure this `goto` should be commented out? If yes, then probably you should remove "Treat it as an error" from comment above.